src/templates/Footer/index.js

```diff
@@ -109,13 +116,13 @@ Footer.propTypes = {
     })
   ),
   /**
-   * Small Footer text.
+   * set the mailto: email address
    */
-  text: PropTypes.string,
+  emailLink: PropTypes.string,
```

Comment:
@petrgazarov This should probably be something like `emailAddress` so that people do think that they should include the whole `mailto:team....`